src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -381,23 +375,24 @@ namespace AirflowNetworkSolver {
         // na
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        int i;
-        int n;
 
         // FLOW:
-        for (n = 1; n <= NetworkNumOfNodes; ++n) {
-            ID(n) = n;
+        for (int i = 1; i <= NetworkNumOfNodes; ++i) {
```

Comment:
Do we have any benefit to redefine "int i=1..." for every for loop?

Reply:
Generally speaking I'd expect the compiler to do the best thing, so there may not be much performance benefit but I wouldn't expect a performance hit either. However, this way the scope where `i` is valid is just in the loop, which protects against unintentional use and clashes.